pkg/cli/cmd/run/run.go

```diff
@@ -159,28 +162,65 @@ func (r *Runner) Run(ctx context.Context) error {
 		return clierrors.Message("Only kubernetes runtimes are supported.")
 	}
 
-	// We start three background jobs and wait for them to complete.
+	applicationLabel, err := labels.NewRequirement(kubernetes.LabelRadiusApplication, selection.Equals, []string{r.ApplicationName})
+	if err != nil {
+		return err
+	}
+
+	applicationSelector := labels.NewSelector().Add(*applicationLabel)
+
+	dashboardNameLabel, err := labels.NewRequirement(kubernetes.LabelName, selection.Equals, []string{"dashboard"})
+	if err != nil {
+		return err
+	}
+
+	dashboardPartOfLabel, err := labels.NewRequirement(kubernetes.LabelPartOf, selection.Equals, []string{"radius"})
+	if err != nil {
+		return err
+	}
+
+	dashboardSelector := labels.NewSelector().Add(*dashboardNameLabel).Add(*dashboardPartOfLabel)
+
+	// We start 5 background jobs and wait for them to complete.
 	group, ctx := errgroup.WithContext(ctx)
 
-	// 1. Display port-forward messages
-	status := make(chan portforward.StatusMessage)
+	// 1. Display port-forward messages for application
+	applicationStatusChan := make(chan portforward.StatusMessage)
 	group.Go(func() error {
-		r.displayPortforwardMessages(status)
+		r.displayPortforwardMessages(applicationStatusChan)
 		return nil
 	})
 
-	// 2. Port-forward
+	// 2. Display port-forward messages for dashboard
+	dashboardStatusChan := make(chan portforward.StatusMessage)
+	group.Go(func() error {
+		r.displayPortforwardMessages(dashboardStatusChan)
+		return nil
+	})
+
+	// 3. Port-forward application
 	group.Go(func() error {
 		return r.Portforward.Run(ctx, portforward.Options{
-			ApplicationName: r.ApplicationName,
-			Namespace:       namespace,
-			KubeContext:     kubeContext,
-			StatusChan:      status,
-			Out:             os.Stdout,
+			LabelSelector: applicationSelector,
+			Namespace:     namespace,
+			KubeContext:   kubeContext,
+			StatusChan:    applicationStatusChan,
+			Out:           os.Stdout,
+		})
+	})
+
+	// 4. Port-forward dashboard
```

Comment:
What happens if the user has opted out of the dashboard in their installation? Does this silently continue without throwing any errors or warnings?

Reply:
just added a commit to handle this :) now we will throw a log message with a docs link:

![image](https://github.com/radius-project/radius/assets/32342549/7fa4fa54-436c-4f89-a6ae-c46a7722fde1)